machinist/mserver/factory.go

```diff
@@ -2,7 +2,7 @@ package mserver
 
 func NewController() (*Controller, error) {
 	en := &Controller{
-		ConnectedNodes: make(map[string][]string),
+		connectedNodes: make(map[string]*Node),
```

Comment:
nit: make is necessary only in a handful of cases, generally recommend using plain initialization when make is not necessary:
```
connectedNodes: map[string]*Node{},
```